tests/playwright/sanity/includes/widgets/tabs.test.js

```diff
@@ -1,10 +1,15 @@
-const { test, expect } = require( '@playwright/test' );
-const WpAdminPage = require( '../../../pages/wp-admin-page.js' );
+import { test, expect } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page.js';
+const EditorPage = require( '../../../pages/editor-page.js' );
+import Content from '../../../pages/elementor-panel-tabs/content.js';
```

Comment:
Can you wrap these tests in a `describe`

Reply:
fixed